src/rpc_payloads.rs

```diff
@@ -37,59 +44,6 @@ pub enum RpcMessage {
     },
 }
 
-impl RpcMessage {
```

Comment:
I guess none of these functions was used anywhere...

Reply:
They were used, but it was kinda stupid. At first they were doing some conversions like `AccountId` to `String`, but especially with the metadata they just became a pass-through that was just really unnecessary. You will see it on the corresponding indexer PR.